assets/queries/cloudFormation/lambda_functions_with_full_privileges/query.rego

```diff
@@ -1,26 +1,6 @@
 package Cx
 
-CxPolicy[result] {
-	resources := input.document[i].Resources
-	resource := resources[name]
-
-	resource.Type == "AWS::Lambda::Function"
-
-	# check if the role referenced in the lambda function properties is defined in the same template
-	fullRole := split(resource.Properties.Role, ".")
-	role := fullRole[0]
-	resources[role]
-
-	policyName := checkPolicies(resources[role].Properties.Policies)
-
-	result := {
-		"documentId": input.document[i].id,
-		"searchKey": sprintf("Resources.%s.Properties.Policies.PolicyDocument", [role]),
-		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("Resources.%s.Properties.Policies[%s].PolicyDocument does not give admin privileges to Resources.%s ", [role, policyName, name]),
-		"keyActualValue": sprintf("Resources.%s.Properties.Policies[%s].PolicyDocument gives admin privileges to Resources.%s ", [role, policyName, name]),
-	}
-}
```

Comment:
Why this rule was removed?

Reply:
Because the difference between both rules was the verification of 'Action' as a string or an array. This "new rule" verifies both cases